bears/generate_package.py

```diff
@@ -40,9 +40,7 @@ def create_file_from_template(template_file, output_file, substitution_dict):
 def create_file_structure_for_packages(root_folder, file_to_copy, object_name):
```

Comment:
Also there seem to be other changes apart from just the `__init__.py` one. Detail everything or break up in to separate commits